demisto_sdk/commands/format/update_layout.py

```diff
@@ -301,100 +308,68 @@ def remove_non_existent_fields_container_layout(self):
         """
         Remove non-existent fields from a container layout.
         """
-        if not self.id_set_file:
-            logger.warning(
-                f"Skipping formatting of non-existent-fields for {self.source_file} as id_set_path argument is missing"
+        if not self.graph:
+            logger.info(
+                f"Skipping formatting of non-existent-fields for {self.source_file} as the no-graph argument was given."
             )
             return
 
-        content_fields = self.get_available_content_fields()
-
         layout_container_items = [
             layout_container_field
             for layout_container_field in LAYOUT_CONTAINER_FIELDS
             if self.data.get(layout_container_field)
         ]
 
+        # get the relevant content item from the graph
+        content_item: ContentItem
+        for content_item in self.graph.search(object_id=self.data.get('id', '')):
+            if content_item.content_type == ContentType.LAYOUT and str(content_item.path) == self.source_file:
+                break
```

Comment:
Consider using `next` instead of the loop-and-break (I'm pretty sure Sourcery suggests it)

Reply:
Sourcery doesn't suggest it, and I don't understand why using next is better. I'll still have to do a for loop and check what is the value returned from next and check if this is the wanted one.

Reply:
See [use-next](https://docs.sourcery.ai/Reference/Python/Default-Rules/use-next/)
It's mostly syntactically cleaner.